resources/model/OpenStudio.idd

```diff
@@ -30404,6 +30404,151 @@ OS:Generator:FuelSupply,
       \minimum 0.0
       \maximum 1.0
 
+OS:Generator:WindTurbine,
+      \memo Wind turbine generator.
+      \min-fields 27
+  A1, \field Handle
+      \type handle
+      \required-field
+  A2, \field Name
+      \required-field
+      \reference GeneratorNames
+  A3, \field Availability Schedule Name
+      \note Availability schedule name for this system. Schedule value > 0 means the system is available.
+      \note If this field is blank, the system is always available.
+      \type object-list
+      \object-list ScheduleNames
```

Comment:
```suggestion
  A3, \field Availability Schedule Name
      \note Availability schedule name for this system. Schedule value > 0 means the system is available.
      \note If this field is blank, the system is always available.
      \type object-list
      \object-list ScheduleNames
      \required-field
```

Set to `model.alwaysOnDiscreteSchedule()` in Ctor:

Reply:
I would **still** make this required-field and default to model.alwaysOnDiscreteSchedule in Ctor even though getter will be optional per Generator.hpp